module/zfs/ddt_zap.c

```diff
@@ -166,3 +166,8 @@ const ddt_ops_t ddt_zap_ops = {
 	ddt_zap_walk,
 	ddt_zap_count,
 };
+
+ZFS_MODULE_PARAM(zfs_dedup, , ddt_zap_leaf_blockshift, INT, ZMOD_RW,
```

Comment:
For regular files the parameters are currently called zfs_default_bs and zfs_default_ibs.  It would be good to be consistent.